HHS/Head-Start-TTADP
# [Prod] Remove fields from the objective form on the RTR

## Description of change
Removes objective sub-fields from the RTR and adjusts the Activity report form accordingly (see attached video)

https://github.com/HHS/Head-Start-TTADP/assets/3288586/60096257-78c4-4dfd-bd86-3dd5b7cc4ccf

PR also includes two data fixes: one for the non-UTF-8 characters that were imported to the courses and the other to save historical root causes on multi-recipient reports, which were not cached originally.

We also add a SQL file to the repository, which can be used to generate a report at a later date.

## How to test
Confirm that:
- Objective fields resources, topics, support type, courses, and files have been removed from the RTR
- On the RTR, text is editable until the objective was used on a report
- On the RTR, goal source is editable until goal is on an approved report
- On the AR, objective text is only editable if created on a report
- On the AR, goal source is editable until goal is on an approved report

## Issue(s)
* https://jira.acf.gov/browse/TTAHUB-2530
* https://jira.acf.gov/browse/TTAHUB-2660
* https://jira.acf.gov/browse/TTAHUB-2560
* https://jira.acf.gov/browse/TTAHUB-2986
* https://jira.acf.gov/browse/TTAHUB-2999

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
